deploy/kubernetes/base/controller.yaml

```diff
@@ -48,6 +48,15 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
+        - name: csi-snapshotter
+          imagePullPolicy: Always
```

Comment:
remove the imagePullPolicy here. Usually it's unnecessary to have to always pull a new image, unless you're reusing the same tag, which is generally only applicable to dev scenarios.

Reply:
Ack